homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -150,6 +158,16 @@ def total_volatile_organic_compounds(self):
         """Return the total volatile organic compounds."""
         return self._total_volatile_organic_compounds
 
+    @property
+    def temperature(self):
+        """Return the total volatile organic compounds."""
```

Comment:
Please update the docstrings for temperature and humidity :)

Reply:
updated the docstrings